python/lsst/ts/standardscripts/auxtel/slew_to_O2_cleanup_position.py

```diff
@@ -0,0 +1,142 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["SlewToO2CleanupPosition"]
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel import ATCS, ATCSUsages
+
+
+class SlewToO2CleanupPosition(salobj.BaseScript):
+    """Put AT in O2 cleanup position.
+
+    This script will slew the auxiliary telescope to the O2 cleanup position.
+    Telescope will be left in O2 cleanup position with tracking disabled.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    """
+
+    def __init__(self, index):
+        super().__init__(
+            index=index,
+            descr="Slew AT to O2 cleanup position.",
+        )
+
+        self.atcs = None
+        self.slew_time_guess = 180
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/slew_to_O2_cleanup_position.yaml
+            title: SlewToO2CleanupPosition v1
+            description: Configuration for SlewToO2CleanupPosition.
+            type: object
+            properties:
+                az:
+                    description: Target Azimuth in degrees.
+                    type: number
+                el:
+                    description: Target Elevation in degrees.
+                    type: number
+                    minimum: 0.0
+                    maximum: 90.0
+                rot_tel:
+                    description: Rotator angle in mount physical coordinates (degrees).
+                    type: number
+                    default: 0.0
+                target_name:
+                    description: Name of the position.
+                    type: string
+                    default: "O2 cleanup position"
+                wait_dome:
+                    description: Wait for dome to be in sync with the telescope?
+                    type: boolean
+                    default: false
+                slew_timeout:
+                    description: Timeout for slew procedure (in seconds).
+                    type: number
+                    default: 180.0
+                ignore:
+                    description: ATCS components to ignore in availability check.
+                    type: array
+                    items:
+                        type: string
+            required:
+                - az
```

Comment:
Instead of making these required, let's add defaults that are sensible. I think for now let's start with az = 0 and el = 20 and I will verify if those are ok later.